### PR TITLE
Let single line command be a single command

### PR DESCRIPTION
**Pull request by 6543**
docker command do not differentiate ...

### CODE REVIEW

File: pipeline/frontend/yaml/types/command.go [hunk 1]
```diff
@@ -16,11 +14,7 @@ type Command strslice.StrSlice
 func (s *Command) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	var stringType string
 	if err := unmarshal(&stringType); err == nil {
-		parts, err := shlex.Split(stringType)
-		if err != nil {
-			return err
-		}
-		*s = parts
+		*s = []string{stringType}
```
Comment (anbraten) [inline pipeline/frontend/yaml/types/command.go:17]: Whats happens to `# ...` commands now? Are they filtered by yaml directly?

Comment (6543) [reply pipeline/frontend/yaml/types/command.go:17]: yes [:-)](https://go.dev/play/p/rTA-Vrj9Zby)

Comment (6543) [reply pipeline/frontend/yaml/types/command.go:17]: -> the yaml nodes do contain them but on serialization they get thrown away

Comment (6543) [reply pipeline/frontend/yaml/types/command.go:17]: ... #1032 will delete it :D

meanwhile get rid of the TODO

**Review by anbraten — APPROVED** [review]

**Pull request merged**